test/TestCases/Expressions/Angle.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+  ==============================================================================
+    DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+
+    Copyright 2023 by sonible GmbH.
+
+    This file is part of VCTR - Versatile Container Templates Reconceptualized.
+
+    VCTR is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License version 3
+    only, as published by the Free Software Foundation.
+
+    VCTR is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License version 3 for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    version 3 along with VCTR.  If not, see <https://www.gnu.org/licenses/>.
+  ==============================================================================
+*/
+
+#include <vctr_test_utils/vctr_test_common.h>
+
+template <std::floating_point T>
+T getAngle (std::complex<T> src) { return std::arg (src); }
```

Comment:
Do you want to replace `std::arg` by by a "discrete" `atan2` statement here, similarly to the `PowerSpectrum` test case?

Reply:
Only did it like that in the other case because there is no `std::power_spectrum` 😉 I actually think that the straightforward standard library functions are a good ground truth for the test cases if they are available